### PR TITLE
Add BuildInfo to ReplIdentity package

### PR DESCRIPTION
**Pull request by lincoln-replit**

### CODE REVIEW

File: api/repl/repl.proto [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  // Snapshots is the GCS bucket where btrfs snapshots are stored.
+  string snapshots = 1;
+
+  // Metadata is the GCS bucket where OT metadata is stored.
+  string metadata = 2;
+}
+
+// This message constitutes the repl metadata and define the repl we're
+// connecting to. All fields are required unless otherwise stated.
+message Repl {
```
Comment (cbrewster) [inline api/repl/repl.proto:18]: Do we need to add the whole Repl message?

**Review by cbrewster — COMMENTED** [review]
Do we intend to gitignore codegened files?

**Comment by lincoln-replit** [comment]
> Do we intend to gitignore codegened files?

I don't think so, we've checked in all the other .pb.go files in https://github.com/replit/go-replidentity/tree/main/api

> Do we need to add the whole Repl message?

We don't need ~95% of the copied contents, I'm just blindly copying to make this repo a little easier to maintain

**Comment by cbrewster** [comment]
Can we only embed the build info message instead of the whole repl token? Just to avoid leaking more internal details here

**Comment by lincoln-replit** [comment]
Yeah sure

**Comment by lincoln-replit** [comment]
@cbrewster Is this more like what you had in mind?

**Review by cbrewster — APPROVED** [review]

**Pull request merged**